fb.c

```diff
@@ -38,6 +38,17 @@ static int vcam_fb_open(struct fb_info *info, int user)
     return 0;
 }
 
+static void swap_in_queue_buffers(struct vcam_in_queue *q)
+{
+    struct vcam_in_buffer *tmp;
+    if (!q)
+        return;
+    tmp = q->pending;
+    q->pending = q->ready;
+    q->ready = tmp;
+    q->pending->filled = 0;
+}
+
```

Comment:
To minimize change, the fb focus on the frame buffer.

Reply:
In the origin version, `videobuf.c` respose for both `vcam_in_q` and `vcam_out_q`, which `control.c`, `fb.c` have dependancy for using function about `vcam_in_q`.

But after refactoring, we only left one function which is related to `vcam_in_q`: `swap_in_queue_buffers`, and only `fb.c` are in use of the function.

So I have rethinked about the purpose of the file relationship. `videobuf.c` can be actually referred to struct `vb2_queue`, which focus only on the `vb2_ops` and `vcam_out_q`.

On the other hand, `fb.c` focus on the framebuffer implementation, and focus on the `vcamfb_ops`. While only `fb` have connection to `vcam_in_q` but not `vb2_queue`, I think it is better to move the function to `fb.c`.

There is another benefit that we can cancel the dependancy between `fb.c` and `videobuf.h`.

Reply:
Great!